### ISSUE TITLE
Can Jeremie Bresson take it over?

### ISSUE THREAD

**Issue by paulvi** [basti1302/startexplorer#67]
@jmini Hi Jeremie

From your comment https://github.com/basti1302/startexplorer/issues/64#issuecomment-140342665 one could see that you may be considering taking this project.

It is up to you how, what and where. I possibly may help a bit. No promises, but I would at least follow.

Could you tell when you can make up your mind, or what stops you.

We should at least say clear status as of now, e.g. "no one" or "waiting Jeremie" or "Jeremie waits help" 
so that no one could think "oh, it seems like someone has already signed up, never mind then"

**Comment by jmini** [comment]
Sorry for the delay…

1/ My personal development time has been dedicated to other pending tasks.
2/ I am still waiting on the opinion of my coworkers to check if we can spend a some company time on this.

----

The discussion is now split over multiple bugs/issues, but If I try to summarize:
* The SmartExplorer will never become a separated Eclipse Project, but the platforms engineers seem to think that all or some features could be integrated into the platform.
* If the features are integrated to in the Platform, they will never be presented like the SmartExplorer does. It is hard to have an “user experience” that fit all needs. This means that we will always need a SmartExplorer plugin that presents the features to the user (menus, shortcuts…). The SmartExplorer plugin can also contain the features that are not integrated to the Platform.

If I take over this plugin, the ownership needs to be to a team in the “eclipselabs” organization. This provides more flexibility to add or remove a maintainer to the team. 

As a first step, we need a rock solid building infrastructure: continuous integration, continuous delivery (from the Git Repo to a nightly P2 update site), otherwise we won’t be able to work on this plugin as a team. 

I did not yet understand if @basti1302 share this vision or not, saying that the current status quo is also okay for him.

**Comment by basti1302** [comment]
@jmini To clarify: I am okay with the status quo *if nothing else works out*. If we can chop StartExplorer up and reuse parts of in Eclipse proper, that would be preferable. So, yes, I do share the vision you outlined.

Btw, somehow the name S**m**artExplorer appeared in the discussion and started to stick around. The original name was S**t**artExplorer. But since the name will probably be dropped in the process, this is not so important.

**Comment by basti1302** [comment]
Closing this in favor of #64
